### PR TITLE
 removing unnecessary if condition in $reverse op of JS code 

### PR DESCRIPTION
**Pull request by alexandrasp**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -238,10 +238,6 @@ operators.$reverse = (template, context) => {
   if (!isArray(value) && !isArray(template['$reverse'])) {
```
Comment (djmitche) [inline src/index.js]: Oh!  I didn't even notice the `if` you removed when I first looked at this code.  I think the `!isArray(template['$reverse'])` here is also redundant.  Is there a situation where `isArray(value)` is false but `isArray(template['$reverse'])` is true?

Comment (alexandrasp) [reply src/index.js]: Yes @djmitche, it makes sense, specially if we check the code...
`let value = render(template['$reverse'], context);`
Calling render that don't change an array to other type. Can it be conclusive? 
I checked the test cases with `$reverse` template...and again always `false && false` or `true&&true`.

Comment (alexandrasp) [reply src/index.js]: I think the code is ready to be merged ;)

Comment (djmitche) [reply src/index.js]: Right, render of an array is always an array.

Comment (djmitche) [reply src/index.js]: Let's get rid of the second part of this conditional too, then merge :)

Comment (alexandrasp) [reply src/index.js]: pushed :)

**Comment by djmitche** [comment]
Thanks, sorry for the back-and-forth :)

**Comment by alexandrasp** [comment]
no problem @djmitche ;)

**Pull request merged**